Description: Scripts need to specify interpreter in the first line
Author: Andreas Tille <tille@debian.org>
Forwarded: not yet
--- a/contrib/shuffleSequences_fasta/shuffleSequences_fasta.py
+++ b/contrib/shuffleSequences_fasta/shuffleSequences_fasta.py
@@ -1,3 +1,4 @@
+#!/usr/bin/python
 import sys
 from Bio import SeqIO
 def interleave(iter1, iter2) :
